regexes/device/mobiles.yml

```diff
@@ -1407,11 +1407,17 @@ Blaupunkt:
 
 # Bmobile
 Bmobile:
-  regex: 'Bmobile[_ ]|AX1075'
+  regex: 'Bmobile[_ ]|AX10(?:[1234567]0|[1345678]5|16|65E|70e|73\+|7[1234]|91)|AX(?:5[12347]0|512|5[123]5|524|6[01256789]0|[67]05|675|68[0135]|680\+|710|745|810|82[013]|92[012])|AX-340|AX7OO'
   device: 'smartphone'
   models:
-    - regex: 'AX1075'
-      model: 'AX1075'
+    - regex: 'AX10([1234567]0|[1345678]5|16|65E|70e|73\+|7[1234]|91)[);/ ]'
+      model: 'AX10$1'
+    - regex: 'AX(5[12347]0|512|5[123]5|524|6[01256789]0|[67]05|675|68[0135]|680\+|710|745|810|82[013]|92[012])[);/ ]'
+      model: 'AX$1'
+    - regex: 'AX-340'
+      model: 'AX340'
+    - regex: 'AX7OO'
+      model: 'AX700'
```

Comment:
Are there any devices of other brands starting with AX? Otherwise we maybe could simplify the whole detections by using `AX-?([1-9][0-9]{2,3})` for all and in the overall match